src/test/java/org/checkstyle/autofix/recipe/UpperEllRecipeTest.java

```diff
@@ -1,40 +1,30 @@
 package org.checkstyle.autofix.recipe;
 
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.openrewrite.java.Assertions.java;
+import java.io.IOException;
 
+import org.checkstyle.autofix.util.AbstractTestUtil;
 import org.junit.jupiter.api.Test;
-import org.openrewrite.test.RecipeSpec;
-import org.openrewrite.test.RewriteTest;
+import org.openrewrite.Recipe;
 
-public class UpperEllRecipeTest implements RewriteTest {
+public class UpperEllRecipeTest extends AbstractTestUtil {
```

Comment:
please rename AbstractTestUtil to AbstractRecipieTest

util, should always mean just collection of independent methods.